src/current/v24.1/rotate-certificates.md

```diff
@@ -36,6 +36,10 @@ You may need to rotate the node, client, or CA certificates in the following sce
 
     This step is application-specific and may require restarting the client.
 
+{{site.data.alerts.callout_info}}
+{% include {{page.version.version}}/misc/cert-auth-using-x509-subject.md %}
```

Comment:
The use of subject fields is not relevant to cert rotation.

Reply:
Removed